gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc

```diff
@@ -4,63 +4,84 @@ include::_attributes.adoc[]
 
-* Linux and Docker: `.yaml` files
+Flex Gateway manages traffic by using different routes with defined rulesets and route order to direct traffic to different sets of upstream services. Each API instance supports multiple routes that can each direct traffic to multiple upstream services.
 
-The following procedures demonstrate applying a simple YAML configuration for an unsecured API with multiple upstream services.
+For Flex Gateways running in Local Mode, API instances with multiple upstream services in are configured by using the `ApiInstance` and the `PolicyBinding` YAML configuration resources. The `ApiInstance` resource defines an API instance, and the `route-weighted` policy resource defines a single route to one or multiple upstream services. To add additional routes, add multiple `route-weighted` resources.
 
-[cols="1,1"]
-|===
-
-|image:install-linux-logo.png[20%,20%,xref="flex-local-publish-api-multiple-services.adoc#publish-an-api-running-behind-flex-gateway-on-linux"]
-
-|image:install-docker-logo.png[25%,25%,xref="flex-local-publish-api-multiple-services.adoc#publish-an-api-running-behind-flex-gateway-in-a-docker-container"]
+The following tutorial uses a preconfigured `route-weighted` policy resource. For information about how to configure the resource, see xref:policies-included-traffic-management.adoc[].
 
-|xref:flex-local-publish-api-multiple-services.adoc#publish-an-api-running-behind-flex-gateway-on-linux[Publish an API Running Behind Flex Gateway on Linux]
-
-|xref:flex-local-publish-api-multiple-services.adoc#publish-an-api-running-behind-flex-gateway-in-a-docker-container[Publish an API Running Behind Flex Gateway in a Docker Container]
+[cols="1a,1a,1a"]
+|===
+|image:install-linux-logo.png[20%,20%,xref="#linux"]
+|image:install-docker-logo.png[25%,25%,xref="docker"]
+|image:install-kubernetes-logo.png[20%,20%,xref="kubernetes"]
 
+|<<linux>>
+|<<docker>>
+|<<kubernetes>>
 |===
 
+[[linux]]
 == Publish an API Running Behind Flex Gateway on Linux 
 
 === Before You Begin
 
 Before getting started, ensure that you have:
 
-* Flex Gateway installed and running in local mode. See xref:flex-install.adoc[] for more information about installing and running the gateway. 
-* Your upstream service URLs. The following example refers to fictional `products-api` and `users-api` services, but you can specify your own API name in `metadata.name` and your service details in `spec.services`.
+* Flex Gateway downloaded. For more information, see xref:flex-install.adoc[]. 
+* Flex Gateway registered and running in Local Mode. See xref:flex-local-reg-run.adoc[Register and Run in Local Mode] for more information. 
+* Your upstream service URLs. The following Publish an API section refers to fictional `products-api` and `products-api-beta` services and a fictional `json-http` API instance. You can specify your own API name in `metadata.name` and your own service details.
```

Comment:
It would be better to put the information about the tutorial where it is needed - in this case, in step 4.